### ISSUE TITLE
schema `default` property is of type string, so doesn't align with the `type` property 

### ISSUE THREAD

**Issue by Mearman** [breadboard-ai/breadboard#1164]

**Comment by paullewis** [comment]
Need a little more on this one. We don't currently support all types (something I'm about to start working on). Which particular type was missing?

**Comment by Mearman** [comment]
this is on the actual TS types. not in the UI

**Comment by dglazkov** [comment]
Yup, this is not great. For @aomarks when considering TS syntax improvements.

**Comment by Mearman** [comment]
here's a good example
https://github.com/breadboard-ai/breadboard/blob/main/packages/breadboard-web/src/boards/open-alex-entity-search-list.ts#L37-L42

**Comment by dglazkov** [comment]
FWIW, there's a decent workaround for now (JSON.stringify), so this is not urgent.
